xled/control.py

```diff
@@ -464,12 +464,8 @@ def update_firmware(self, stage0, stage1):
                 assert False
 
         if fw_stage_sums != uploaded_stage_sums:
-            log.error(
-                "Firmware SHA1SUMs: %r != uploaded SHA1SUMs",
-                fw_stage_sums,
-                uploaded_stage_sums,
-            )
-            msg = "Firmware SHA1SUMs doesn't match to uploaded SHA1SUMs.".format(stage)
+            log.error("Firmware SHA1SUMs: %r != uploaded SHA1SUMs", fw_stage_sums)
```

Comment:
Why this change?

Reply:
The second variable wasn't being printed.

Reply:
Ah right, good catch!